### ISSUE TITLE
Rainway streams very tiny on client

### ISSUE THREAD

**Issue by alpha15x** [RainwayApp/bug-tracker#173]
So, I happened to purchase a new monitor recently and set it up as my main monitor for the host computer. Rainway launches and is configurable, although I did have to change my display over to the new monitor. The only differences with this monitor is that it is a 21:9 aspect ratio because it is an ultrawide, and it is running at a resolution of 3440x1440 because of this. My previous monitor was the standard 16:9 and standard 1920x1080 and everything scaled properly. The only thing I could think is that Rainway does not play well with ultrawide resolutions?

I did try to set my stream scaling to the client's native resolution of 720p, but regardless of what setting I set it to, it does not change the size. 

I'm wondering if anyone else is using an ultrawide monitor that can give me an insight of the problem?

**Comment by Jessomadic** [comment]
Turn scaling off 100% and see if that helps at all. Let me know if it does not

**Comment by alpha15x** [comment]
Found out the problem.

Each time that I happen to make a change with Rainway, I'd basically have to fully close out of it and relaunch it afterwards for it to take effect. I had tried just keeping it up after I made changes but that seemed to be why.

Disabling the scaling did fix the issue, although a set of widescreen bars are now at the top and bottom of the stream. I believe this is to compensate streaming from a 21:9 source to a 16:9 display.
